build.gradle

```diff
@@ -55,7 +56,7 @@ publishing {
             from components.java
             groupId 'com.datadoghq'
             artifactId 'datadog-lambda-java'
-            version '0.0.5'
+            version version
```

Comment:
I could have sworn I tried this at one point ant it didn't work. I'm very glad you made it work! 😄

Reply:
Woo!